api/RTCPeerConnection.json

```diff
@@ -0,0 +1,1420 @@
+{
+  "api": {
+    "RTCPeerConnection": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/RTCPeerConnection",
+        "support": {
+          "webview_android": [
+            {
+              "prefix": "webkit",
+              "version_added": null
+            },
+            {
+              "notes": "unprefixed",
+              "version_added": "56"
+            }
+          ],
+          "chrome": {
+            "version_added": true
+          },
+          "chrome_android": [
+            {
+              "prefix": "webkit",
+              "version_added": null
+            },
+            {
+              "notes": "unprefixed",
+              "version_added": "56"
+            }
+          ],
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "22"
+          },
+          "firefox_android": [
+            {
+              "prefix": "moz",
+              "version_added": "22"
+            },
+            {
+              "notes": "unprefixed",
+              "version_added": "44"
+            }
+          ],
+          "ie": {
+            "version_added": null
+          },
+          "opera": {
+            "version_added": null
+          },
+          "opera_android": {
+            "version_added": null
```

Comment:
Both versions of Opera are 43 for unprefixed. https://www.chromestatus.com/features/5312661013135360